src/core/comparable.ml

```diff
@@ -5,3 +5,9 @@ module type S = sig
 
   val compare:t Comparator.t
 end
+
+module type C = sig
```

Comment:
Hi thanks for the review, sorry I was somehow busy today and just had the chance to look at it. Could you elaborate on the design if we have modular explicit? I am just curious how it would look like and why it would affect adding this interface. Thanks